assets/js/blocks/mini-cart/utils/data.ts

```diff
@@ -128,15 +129,9 @@ export const getMiniCartTotalsFromLocalStorage = ():
 export const getMiniCartTotalsFromServer = async (): Promise<
 	[ CartResponseTotals, number ] | undefined
 > => {
-	return fetch( '/wp-json/wc/store/v1/cart/' )
-		.then( ( response ) => {
-			// Check if the response was successful.
-			if ( ! response.ok ) {
-				throw new Error();
-			}
-
-			return response.json();
-		} )
+	return apiFetch< CartResponse >( {
```

Comment:
@dinhtungdu Thank you for the fix! Just for my learning, how apiFetch is different than browser fetch?

In the [docs](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-api-fetch/), it says `It’s a wrapper around window.fetch.`

Reply:
@tarunvijwani It comes with some middleware, one of them is `createRootURLMiddleware` which WP uses to set the correct root API URL in `wp_default_packages_inline_scripts()`:
```
	$scripts->add_inline_script(
		'wp-api-fetch',
		sprintf(
			'wp.apiFetch.use( wp.apiFetch.createRootURLMiddleware( "%s" ) );',
			sanitize_url( get_rest_url() )
		),
		'after'
	);
```

So by using `apiFetch`, we can always fetch the correct endpoint by just providing the path.

Reply:
That's really helpful. Thank you!